india_banking_connector/host_to_host/doctype/citi_h2h_connector/citi_h2h_connector.py

```
@@ -0,0 +1,582 @@
+					frappe.throw(
+						"Encryption failed please check the logs for more details"
+					)
+
+		if encrypted_file_urls:
+			frappe.db.set_value("H2H Payment Log", log_id, encrypted_file_urls)
+			return True
+
+		return False
+
+	def init_gpg(self):
+		gpg_home = "/tmp/.gnupg_citi_h2h/"
```

Ruff (0.14.14)

[error] Probable insecure usage of temporary file or directory: "/tmp/.gnupg_citi_h2h/"

(S108)